apps/leo_gateway/priv/leo_gateway.conf

```diff
@@ -109,6 +109,16 @@ bucket_prop_sync_interval = 300
 ## wtperf is set to the same value.
 ## nfs.nfsd.wtmax = 5242880
 
+## NFSd's Readdir Entry Table Scan Interval in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_scan_int = 1800
+
+## NFSd's Readdir Entry TTL in seconds (leo-project/leofs#612)
+## nfs.nfsd.readdir_entry_ttl = 3600
+
+## NFSd's Readdir Entry Table Memory Scanning Threshold in bytes (leo-project/leofs#612)
+## nfs.nfsd.readdir_mem_thres = 268435456
+
```

Comment:
reference: http://stackoverflow.com/questions/21973760/how-to-identify-the-exact-memory-size-of-an-ets-table